src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -548,7 +589,7 @@ private void initClassTreeMap(List<ColumnFamilyDescriptor> columnFamilyDescripto
         classTreeMap = MapUtils.unmodifiableMap(classMap);
     }
 
-    private DBOptions createOptions(String logPath, String configFile) throws IOException {
+    private DBOptions createOptions(String logPath, String configFile, List<ColumnFamilyDescriptor> columnFamilyDescriptors) throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'createOptions(String, String, List)' has a cyclomatic complexity of 16.](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4453821)